nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -444,6 +444,9 @@ def _impl(inputs, attr, params):
 def _sum():
     def _impl(inputs, attr, params):
         axis = params.pop(inputs[1].list_output_names()[0]).asnumpy()
+        # convert to tuple for preventing invalid parameter format error
+        if isinstance(axis, np.ndarray):
```

Comment:
This check should be redundant since `NDArray::asnumpy` always returns np.ndarray. Other operations use code  `axis.asnumpy()[0]` in similar places. I suggest using same approach here.

Reply:
Thnks for your suggestion.

>This check should be redundant since NDArray::asnumpy always returns np.ndarray

You are right. I'll fix it.

> axis.asnumpy()[0]

I think that this approach will not work well when the sum operator has multiple axes (`axis = None` will be full axes) because this always gets only the first axis from the array of axes.

Reply:
> axis.asnumpy()[0]

Yes, I understand now. OK.